go-controller/pkg/ovn/policy_test.go

```diff
@@ -2311,3 +2356,43 @@ var _ = ginkgo.Describe("OVN NetworkPolicy Low-Level Operations", func() {
 		gomega.Expect(gp.delNamespaceAddressSet(four.GetObjectID(libovsdbops.ObjectNameKey))).To(gomega.BeFalse())
 	})
 })
+
```

Comment:
nit: FWIW in EIP test we put UTILs at the bottom, but in this file they seem to be at the top, I won't block on this accord but in the future putting it here could lead to duplication of utils given ppl might look  for a set at the start and then not notice these below here..I'd keep them consistent together in one place (near the existing address set utils)

Reply:
@JacobTanenbaum ^ PTAL

Reply:
moved the functions with the other test utils